modules/configuring-logging-collector.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploying.adoc
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-logging-collector_{context}"]
+= Configuring the log collector
+
+You can configure which log collector type your {logging} uses by modifying the `ClusterLogging` custom resource (CR).
```

Comment:
We should consider qualifying this in context of mCLF ?

Reply:
I think let's open a follow up Jira for that, I want to keep the scope to structural changes as much as possible for now so that we can get the restructure finished. Does that work for you?

Reply:
Expanded https://issues.redhat.com/browse/OBSDOCS-612 to include updating ClusterLogging CRs to talk about mCLF in the scope